### PR TITLE
matrix function edit

### PR DESCRIPTION
**Pull request by osherboudara99**

### CODE REVIEW

File: Dijkstras-Algo.ipynb [hunk 1]
```diff
@@ -229,30 +229,31 @@
...
+    "                if (A[j][k] > 0) and (visited[k] == False) and (nodes[k]['dist'] > nodes[j]['dist'] + A[j][k]): \n",
+    "                    nodes[k]['dist'] = nodes[j]['dist'] + A[j][k]\n",
+    "                    nodes[k]['prev'] = j\n",
+    "        return nodes\n",
     "        \"\"\"                   \n",
     "        print (\"Graph Vertex \\t Distance from Source\") \n",
     "        print(\"-------------------------------------\")\n",
     "        for node in range(len(A)): \n",
-    "            print (node, \"\\t\\t\", distance[node]) \n",
+    "            print (node, \"\\t\\t\", nodes[node]) \n",
```
Comment (eytan-ohana) [inline Dijkstras-Algo.ipynb]: can you remove the space between print and (

**Comment by eytan-ohana** [comment]
@osherboudara99 it looks good :+1:  
* can you use the function i made to display a path between your sources and some other node in the graph.
* for directed paths or nodes where there is no path can we add some sort of way to indicate this
      - maybe we need to change the shortest path function or even the output of dijkstra matrix/list
* and dont forget the merge conflict

**Comment by osherboudara99** [comment]
* initializing prev in node dict to None and source vertex prev initialized to source vertex
* use shortest path function after each graph
* create function for path of source to specified node
* create function to print node dictionary in table 
* resolve merge conflict

**Comment by eytan-ohana** [comment]
Closing since there was an error with git fixed and merged locally.

**Pull request closed without merging**